### PR TITLE
Move jWiki to this monorepo

### LINKED ISSUE #12: Move jWiki to this monorepo

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
## Description
Closes #12.

Copied from jWiki(https://github.com/padaiyal/jWiki).
The jWiki repository will be archived.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [x] **All classes have documentation comments.**
- [x] **All methods have documentation comments.**
- [ ] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why. - NA as this is more of a wiki than a library.
- [ ] **All information to be logged/displayed to the user are internationalized.** If not, explain why. - NA as this is more of a wiki than a library.
- [x] **README.md has been updated if needed.**
- [x] **pom.xml version is set to the latest date** If not, explain why.

## Assignee's Checklist
- [x] **All GitHub action checks have passed.**
- [ ] **All reviewers have approved.**
- [x] **Relevant documentation has been updated if needed.**
- [x] **pom.xml version is set to the latest date** If not, explain why.

### CODE REVIEW

File: libs/commandline/README.md [hunk 1]
```diff
@@ -52,7 +52,10 @@ Add the following dependency tag to the pom.xml of the dependant project:
 <dependency>
     <groupId>io.github.padaiyal.libs</groupId>
     <artifactId>commandline</artifactId>
+<<<<<<< HEAD
```
Comment (abprash) [inline libs/commandline/README.md:55]: Looks like this merge conflict needs to be resolved.

File: libs/i18n/README.md [hunk 2]
```diff
@@ -52,7 +52,10 @@ Add the following dependency tag to the pom.xml of the dependant project:
 <dependency>
     <groupId>io.github.padaiyal.libs</groupId>
     <artifactId>i18n</artifactId>
+<<<<<<< HEAD
```
Comment (abprash) [inline libs/i18n/README.md:55]: same as above?

File: libs/vaidhiyar/README.md [hunk 3]
```diff
@@ -57,7 +57,10 @@ Add the following dependency tag to the pom.xml of the dependant project:
 <dependency>
     <groupId>io.github.padaiyal.libs</groupId>
     <artifactId>vaidhiyar</artifactId>
+<<<<<<< HEAD
```
Comment (abprash) [inline libs/vaidhiyar/README.md:60]: same as above?

**Comment by abprash** [comment]
Does this file also need updating ?
https://github.com/padaiyal/jMonocle/blob/main/libs/filesystem/README.md?plain=1#L63

File: libs/commandline/README.md (hunk 1 above)
Comment (the-c0d3br34k3r) [reply libs/commandline/README.md:55]: Good catch! Resolved.

File: libs/vaidhiyar/README.md (hunk 3 above)
Comment (the-c0d3br34k3r) [reply libs/vaidhiyar/README.md]: Resolved.

**Comment by the-c0d3br34k3r** [comment]
> Does this file also need updating ? https://github.com/padaiyal/jMonocle/blob/main/libs/filesystem/README.md?plain=1#L63

Which part in specific?

File: libs/i18n/README.md (hunk 2 above)
Comment (the-c0d3br34k3r) [reply libs/i18n/README.md]: Resolved.

**Comment by abprash** [comment]
> > Does this file also need updating ? https://github.com/padaiyal/jMonocle/blob/main/libs/filesystem/README.md?plain=1#L63
> 
> Which part in specific?

The link mentions line 63. I'd thought the `version` attribute needed to be updated.

File: libs/commandline/README.md (hunk 1 above)
Comment (abprash) [reply libs/commandline/README.md]: Nice, thanks.

**Comment by the-c0d3br34k3r** [comment]
> > > Does this file also need updating ? https://github.com/padaiyal/jMonocle/blob/main/libs/filesystem/README.md?plain=1#L63
> > 
> > 
> > Which part in specific?
> 
> The link mentions line 63. I'd thought the `version` attribute needed to be updated. Similar to the other pom updates.

Good point. I've updated it. Thanks!

**Review by abprash — APPROVED** [review]
Looks good! 🚀

**Pull request merged**